draft-ietf-tls-tls13.md

```diff
@@ -1288,7 +1298,8 @@ At this point, the handshake is complete, and the client and server
 derive the keying material required by the record layer to exchange
 application-layer data protected through authenticated encryption.
 Application data MUST NOT be sent prior to sending the Finished message and
-until the record layer starts using encryption keys.
+until the record layer starts using encryption keys, except as specified
+in \[{{zero-rtt-data}}].
```

Comment:
I think we've talked about this statement before, so feel free to tell me that it's already decided to keep it this way -- I don't have any new arguments to add.